src/gmusic-theme.js

```diff
@@ -44,20 +52,40 @@ window.GMusicTheme = class GMusicTheme {
   }
 
   /**
-   * Enables the custom theme
-   */
+  * Enabled the dark theme, this allows for backwards compatibility
+  */
   enable() {
-    document.body.classList.add(CONSTANTS.CLASS_NAMESPACE);
-    this.enabled = true;
-    this._drawLogo();
+    this.enableAll();
+  }
+
+  /**
+   * Enables the custom theme in dark mode (All colors)
+   */
+  enableAll() {
+    this.disable();
+    document.querySelector('html').classList.add(CONSTANTS.CLASS_NAMESPACE);
```

Comment:
Instead of using `querySelector` for `html`, you can use `document.documentElement`